### PR TITLE
Add files that don't match the System.*/Microsoft.* pattern to list of files to sign.

### PR DESCRIPTION
**Pull request by crummel**
Fixes clrcompression.dll and SOS.NETCore.dll not being signed.

### CODE REVIEW

**Comment by crummel** [comment]
@dagood @gkhanna79 @weshaggard @livarcocc

**Comment by weshaggard** [comment]
How did you find this set? Did you do signing check on everything in the shared framework?

File: build_projects/dotnet-host-build/sign.proj [hunk 1]
```diff
@@ -44,9 +47,24 @@
...
+      <FilesToSign Include="$(IntermediateOutputPath)sharedFrameworkPublish/shared/**/mscorlib.dll">
+        <Authenticode>$(CertificateId)</Authenticode>
+      </FilesToSign>
+      <FilesToSign Include="$(IntermediateOutputPath)sharedFrameworkPublish/shared/**/netstandard.dll">
+        <Authenticode>$(CertificateId)</Authenticode>
+      </FilesToSign>
+      <FilesToSign Include="$(IntermediateOutputPath)sharedFrameworkPublish/shared/**/SOS.NETCore.dll">
+        <Authenticode>$(CertificateId)</Authenticode>
+      </FilesToSign>
+      <FilesToSign Include="$(IntermediateOutputPath)sharedFrameworkPublish/shared/**/System.dll">
```
Comment (weshaggard) [inline build_projects/dotnet-host-build/sign.proj]: Do we even have a System.dll in here?

File: build_projects/dotnet-host-build/sign.proj [hunk 2]
```diff
@@ -44,9 +47,24 @@
 
   <Target Name="GetSignSharedFrameworkCrossgenedAssembliesFiles">
     <ItemGroup>
+      <FilesToSign Include="$(IntermediateOutputPath)sharedFrameworkPublish/shared/**/clrcompression.dll">
+        <Authenticode>$(CertificateId)</Authenticode>
+      </FilesToSign>
       <FilesToSign Include="$(IntermediateOutputPath)sharedFrameworkPublish/shared/**/Microsoft.*.dll">
         <Authenticode>$(CertificateId)</Authenticode>
       </FilesToSign>
+      <FilesToSign Include="$(IntermediateOutputPath)sharedFrameworkPublish/shared/**/mscorlib.dll">
```
Comment (weshaggard) [inline build_projects/dotnet-host-build/sign.proj]: We don't have mscorlib, netstandard, or System.dll in 1.1.

**Comment by crummel** [comment]
I just copied from the 2.0 file list.  I cut it down to just clrcompression, Microsoft.*.dll, SOS.NETCore.dll, System.*.dll.  We can see if @livarcocc has the full signcheck results before we finalize the list.

**Comment by crummel** [comment]
This is the list I get, should any of these others be signed?
shared\Microsoft.NETCore.App\1.0.5\corehost.exe
shared\Microsoft.NETCore.App\1.0.5\dotnet.exe
shared\Microsoft.NETCore.App\1.1.2\corehost.exe
shared\Microsoft.NETCore.App\1.1.2\dotnet.exe
sdk\1.0.4-rc4-004988\cs\vstest.console.resources.dll
sdk\1.0.4-rc4-004988\de\vstest.console.resources.dll
sdk\1.0.4-rc4-004988\es\vstest.console.resources.dll
sdk\1.0.4-rc4-004988\fr\vstest.console.resources.dll
sdk\1.0.4-rc4-004988\it\vstest.console.resources.dll
sdk\1.0.4-rc4-004988\ja\vstest.console.resources.dll
sdk\1.0.4-rc4-004988\ko\vstest.console.resources.dll
sdk\1.0.4-rc4-004988\pl\vstest.console.resources.dll
sdk\1.0.4-rc4-004988\pt-BR\vstest.console.resources.dll
sdk\1.0.4-rc4-004988\ru\vstest.console.resources.dll
sdk\1.0.4-rc4-004988\tr\vstest.console.resources.dll
sdk\1.0.4-rc4-004988\zh-Hans\vstest.console.resources.dll
sdk\1.0.4-rc4-004988\zh-Hant\vstest.console.resources.dll
shared\Microsoft.NETCore.App\1.1.2\clrcompression.dll
shared\Microsoft.NETCore.App\1.1.2\SOS.NETCore.dll

File: build_projects/dotnet-host-build/sign.proj [hunk 3]
```diff
@@ -35,6 +35,9 @@
       <FilesToSign Include="$(OutDir)corehost/**/hostpolicy.dll">
         <Authenticode>$(CertificateId)</Authenticode>
       </FilesToSign>
+      <FilesToSign Include="$(OutDir)corehost/**/dotnet.exe">
```
Comment (weshaggard) [inline build_projects/dotnet-host-build/sign.proj]: corehost.exe as well?

**Comment by gkhanna79** [comment]
corhost and dotnet should **not** be signed for 1.0.x and 1.1.x releases at all.

File: build_projects/dotnet-host-build/sign.proj (hunk 3 above)
Comment (gkhanna79) [inline build_projects/dotnet-host-build/sign.proj]: Neither corehost nor dotnet should be signed for 1.0.x and 1.1.x.

Comment (crummel) [inline build_projects/dotnet-host-build/sign.proj]: I removed dotnet.exe from the list.

**Comment by crummel** [comment]
Then all we have left unsigned is the resource DLLs, and that's expected if I remember correctly?

**Comment by livarcocc** [comment]
I believe so. Kick off a dummy build on the CLI once new signed bits are generated and I will try the VS insertion to make sure there is nothing missing.

**Comment by weshaggard** [comment]
>Then all we have left unsigned is the resource DLLs, and that's expected if I remember correctly?

The resource dll's are part of the shared framework anyway. So I don't think we need to do anything

**Comment by gkhanna79** [comment]
@crummel Are you planning to port this change to release/1.0.0 branch as well?

**Pull request merged**